src/Styleguide/Components/StepSummaryItem.tsx

```diff
@@ -0,0 +1,45 @@
+import { Sans, Serif, space } from "@artsy/palette"
+import React from "react"
+import styled from "styled-components"
+import { BorderBox } from "Styleguide/Elements/Box"
+import { Flex, FlexProps } from "Styleguide/Elements/Flex"
+import { media } from "Styleguide/Elements/Grid"
+
+interface StepSummaryItemProps extends FlexProps {
+  title: string
+  onChange: () => void
+  children: React.ReactNode | null
```

Comment:
Just out of curiosity, this only removes the optional typing, right? Why is that needed/helpful?

Reply:
`children` isn't in FlexProps, and `StepSummaryItem` is a SFC so doesn't get `children` automatically. If I delete this:

![no children](https://user-images.githubusercontent.com/1242537/43476421-8d598d00-94c6-11e8-8ef8-487a388eef30.gif)

Reply:
Ah I see. The issue is that the SFC is not typed accordingly. If you were to do this then I think it should work:

```tsx
export const StepSummaryItem: React.SFC<Props> = (...) => {...}
```

Reply:
Done :+1: This is a neat pattern.